app/Actions/CreateDatabase.php

```diff
@@ -2,10 +2,68 @@
 
 namespace App\Actions;
 
+use App\Shell\Shell;
+use Illuminate\Support\Str;
+use Symfony\Component\Process\ExecutableFinder;
+
 class CreateDatabase
 {
+    use LamboAction;
+
+    protected $finder;
+    protected $shell;
+
+    public function __construct(Shell $shell, ExecutableFinder $finder)
+    {
+        $this->finder = $finder;
+        $this->shell = $shell;
+    }
+
     public function __invoke()
     {
-        // @todo
+        if (! config('lambo.store.create_database')) {
+            return;
+        }
+
+        if (! $this->mysqlExists()) {
+            return "MySql does not seem to be installed. Skipping new database creation.";
+        }
+
+        $this->logStep('Creating database');
+        $process = $this->shell->execInProject($this->command());
+        $this->abortIf(! $process->isSuccessful(), "The new database was not created.", $process);
+
+        return 'Created a new database ' . config('lambo.store.database_name');
+    }
+
+    protected function mysqlExists()
+    {
+        return $this->finder->find('mysql') !== null;
+    }
+
+    protected function command()
+    {
+        $this->checkDatabaseName();
+
+        return sprintf(
+            'mysql --user=%s --password=%s -e "CREATE DATABASE IF NOT EXISTS %s";',
+            config('lambo.store.database_username'),
+            config('lambo.store.database_password'),
+            config('lambo.store.database_name')
+        );
+    }
+
+    protected function checkDatabaseName()
+    {
+        $configuredDatabaseName = config('lambo.store.database_name');
+
+        if (! Str::contains($configuredDatabaseName, '-')) {
+            return $configuredDatabaseName;
+        }
+
+        config()->set(['lambo.store.database_name' => str_replace('-', '_', $configuredDatabaseName)]);
```

Comment:
What if we handle this once, in the initialization, so it's not duplicated between this command and CustomizeDotEnv?

Reply:
Done.